src/source/Signaling/Signaling.c

```diff
@@ -434,12 +461,14 @@ STATUS refreshIceConfigurationCallback(UINT32 timerId, UINT64 scheduledTime, UIN
 
     DLOGD("Refreshing the ICE Server Configuration");
 
-    // Check if we are in a connected state and if not bail
+    // Check if we are in a connected state or ready state and if not bail.
+    // The ICE state will be called in any other states
     CHK_STATUS(getStateMachineCurrentState(pSignalingClient->pStateMachine, &pStateMachineState));
-    CHK(pStateMachineState->state == SIGNALING_STATE_CONNECTED, retStatus);
+    CHK(pStateMachineState->state == SIGNALING_STATE_CONNECTED ||
+        pStateMachineState->state == SIGNALING_STATE_READY, retStatus);
 
     // Force terminate the connection for now as LWS doesn't allow parallel processing
```

Comment:
is this comment still valid?

Reply:
D'oh - will change the comment. Thanks for pointing out